### PR TITLE
Fix `print_stats` function for RAMSES datasets.

### PR DESCRIPTION
**Pull request by cphyc**
## PR Summary

The `print_stats` function is broken for RAMSES datasets.

## PR Checklist

- [x] Adds a test for any bugs fixed. Adds tests for new features.

### CODE REVIEW

**Comment by neutrinoceros** [comment]
I'm not sure the new test is actually preventing this kind of bug to reappear. Can you add a note we should use capsys to actually test the output (when pytest is our only test runner) ?

File: yt/frontends/ramses/tests/test_outputs.py [hunk 1]
```diff
@@ -638,7 +638,7 @@ def test_max_level():
         assert any(ds.r["index", "grid_level"] == 2)
 
 
-@requires_file(ramses_new_format)
+@requires_file(output_00080)
```
Comment (munkm) [inline yt/frontends/ramses/tests/test_outputs.py:641]: Good correction here!!!

File: yt/frontends/ramses/tests/test_outputs.py [hunk 2]
```diff
@@ -658,3 +658,11 @@ def test_invalid_max_level():
     for lvl, convention in invalid_type_args:
         with assert_raises(TypeError):
             yt.load(output_00080, max_level=lvl, max_level_convention=convention)
+
+
+@requires_file(ramses_new_format)
+def test_print_stats():
+    ds = yt.load(ramses_new_format)
+
+    # Should work
```
Comment (munkm) [inline yt/frontends/ramses/tests/test_outputs.py:667]: 🤞

**Review by munkm — APPROVED** [review]
I think this looks good!

**Comment by cphyc** [comment]
I should have written it more clearly, but when I say broken I mean it raises an exception.

**Review by neutrinoceros — APPROVED** [review]

**Review by neutrinoceros — APPROVED** [review]

**Pull request merged**